kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js

```diff
@@ -1,114 +1,111 @@
-define([
-    'bluebird',
-    'kb_common/html',
-    'common/ui',
-    'util/jobLogViewer',
-    './jobStateList'
-], function (
+define(['bluebird', 'kb_common/html', 'common/ui', 'util/jobLogViewer', './jobStateList'], (
     Promise,
     html,
     UI,
-    LogViewer,
+    JobLogViewer,
```

Comment:
Renamed to use `JobLogViewer` throughout the codebase (in another file, it is referred to as `JobViewer` 🤦 ).